packages/central-server/src/apiV2/GETProjects.js

```diff
@@ -39,14 +39,62 @@ export class GETProjects extends GETHandler {
     entity_hierarchy: ['entity_hierarchy.id', 'project.entity_hierarchy_id'],
   };
 
+  countriesFieldKey = 'countries';
+
   async findSingleRecord(projectId, options) {
     const projectPermissionChecker = accessPolicy =>
       assertProjectPermissions(accessPolicy, this.models, projectId);
     await this.assertPermissions(
       assertAnyPermissions([assertBESAdminAccess, projectPermissionChecker]),
     );
 
-    return super.findSingleRecord(projectId, options);
+    const countryNameKey = options.columns.find(x => Object.keys(x)[0] === this.countriesFieldKey);
+    if (!countryNameKey) {
+      return super.findSingleRecord(projectId, options);
+    }
+
+    const columns = options.columns.filter(x => Object.keys(x)[0] !== this.countriesFieldKey);
+    const dbOptions = { ...options, columns: [...columns, { id: 'project.id' }] };
+    const countriesById = await this.getProjectCountriesByProjectId();
```

Comment:
Getting all countries for all projects when only requesting a single record seems excessive? Maybe we can split out a singular version of the `getProjectCountries` function